AutomatedTesting/Gem/PythonTests/AWS/CMakeLists.txt

```diff
@@ -12,12 +12,6 @@
 ################################################################################
 
 if(PAL_TRAIT_BUILD_TESTS_SUPPORTED AND PAL_TRAIT_BUILD_HOST_TOOLS)
-    # Only enable AWS automated tests on Windows
-    set(SUPPORTED_PLATFORMS "Windows" "Linux")
-    if (NOT "${PAL_PLATFORM_NAME}" IN_LIST SUPPORTED_PLATFORMS)
```

Comment:
Is there any danger of the tests running on Mac, Mobile etc with this removed?

Reply:
Mobile shouldn't be an concern since we check the PAL_TRAIT_BUILD_HOST_TOOLS flag and mobile platforms will be ignored. These tests will run on Mac with this change but I think we do want to the platform is supported. We don't define the jobs for Mac and it won't run during the nightly build.